Resources/doc/reference/field_types.rst

```diff
@@ -118,6 +118,7 @@ Parameter                               Description
 ======================================  ==================================================================
 **hide_protocol**                       remove protocol part from the link text
 **url**                                 URL address (e.g. ``http://example.com``)
+**attributes**                          array of html tag attributes (e.g. ``array('target' => '_blank')``)
```

Comment:
Can you add an example in the code section below? Would be nice to just modify one of the examples that are there already